cmd/main.go

```diff
@@ -45,6 +48,8 @@ func main() {
 		log.Fatal(err)
 	}
 
+	server.AddHandler("HCS_TOPIC_MSG", cmh.NewConsensusMessageHandler(*messageRepository))
```

Comment:
One general feedback that I have is that the `messageTypes` of the handlers/watchers are hardcoded in different places. 
We ideally want to have them configured/hardcoded into one place only.
`HCS_TOPIC_MSG` is hardcoded here and in `consensus-message.go`.

One suggestion is to define the message type in the watcher (as it is now) and read it from there in the main where you attach the handlers